apps/web/src/components/terminal/TerminalSearchBar.tsx

```
@@ -0,0 +1,113 @@
+        onChange={e => setQuery(e.target.value)}
+        placeholder="Find..."
+        className="bg-transparent text-foreground text-sm w-40 outline-none placeholder:text-muted-foreground"
+      />
+
+      <button
+        onClick={() => setCaseSensitive(v => !v)}
+        className={`p-0.5 rounded hover:bg-muted transition-colors ${
+          caseSensitive ? 'text-primary bg-muted' : 'text-muted-foreground'
+        }`}
+        title="Case Sensitive"
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)